procrustes/psdp.py

```diff
@@ -31,8 +31,164 @@
 import scipy
 from scipy.optimize import minimize
 
+__all__ = ["psdp_woodgate", "psdp_peng"]
```

Comment:
If we put 
```python
__all__ = [
    "psdp_woodgate", 
    "psdp_peng",
]
```
which is more maintainable.

Reply:
fixed in [171a794](https://github.com/theochem/procrustes/pull/180/commits/171a794f55ef10c0a2f5ad17240fdb094fd10bef)